.travis.yml

```diff
@@ -14,16 +14,11 @@ matrix:
     - jdk: openjdk9
       script: ./mvnw install
     - jdk: oraclejdk10
-      # XXX: On JDK 10 Error Prone causes warnings to be emitted; see
-      # https://github.com/google/error-prone/issues/860.
-      script: ./mvnw install -Dverification.warn
+      script: ./mvnw install
     - jdk: openjdk10
-      # XXX: On JDK 10 Error Prone causes warnings to be emitted; see
-      # https://github.com/google/error-prone/issues/860.
-      script: ./mvnw install -Dverification.warn
+      script: ./mvnw install
     - jdk: openjdk-ea
```

Comment:
Should we include `oraclejdk-ea`, too?

It also _seems_  that `openjdk11` is now available. (https://docs.travis-ci.com/user/languages/java/#using-java-10-and-later)

Reply:
> Should we include `oraclejdk-ea`, too?

Well, see #30 and travis-ci/travis-ci#9963 ;)

> It also _seems_  that `openjdk11` is now available.

Good one; will try that.

Reply:
Just [tested](https://travis-ci.org/PicnicSupermarket/reactive-support/builds/427399709); `oraclejdk-ea` still fails. Added `oraclejdk11` and `openjdk11` :)